integration-enabler-java/src/main/java/com/ca/mfaas/eurekaservice/client/config/ApiMediationServiceConfig.java

```diff
@@ -19,22 +19,192 @@
+     *
+     *          http://gateway:port/api/v1/vantageprod1/endpoint1/...
+     *
+     *          </li>
+     *      </ul>
+     *    XML Path: /instance/app
+     */
     private String serviceId;
+
+    /**
+     * * **title** (XML Path: /instance/metadata/apiml.service.title)
+     *
+     *   This parameter specifies the human readable name of the API service instance.
+     *
+     *   **Examples:**
+     *
+     *   `Endevor Prod` or `Sysview LPAR1`
+     *
+     *   This value is displayed in the API Catalog when a specific API service instance is selected.
+     *   This parameter can be externalized and set by the customer system administrator.
+     *
+     *   **Tip:** We recommend that service developer provides a default value of the `title`.
+     *         Use a title that describes the service instance so that the end user knows the specific purpose of the service instance.
+     *
+     */
     private String title;
+
+    /**
+     * * **description** (XML Path: /instance/metadata/apiml.service.description)
+     *
+     *     This parameter specifies a short description of the API service.
+     *
+     *     **Examples:**
+     *
+     *     `CA Endevor SCM - Production Instance` or `CA SYSVIEW running on LPAR1`
+     *
+     *      This value is displayed in the API Catalog when a specific API service instance is selected.
+     *      This parameter can be externalized and set by the customer system administrator.
+     *
+     *   **Tip:** Describe the service so that the end user understands the function of the service.
+     */
     private String description;
+
+    /**
+     *     Specifies the base URL pointing to your service.
+     *
+     *     In _XML_ configuration, the baseUrl is decomposed into the following basic URL parts: `hostname`, `ipAddress` and `port` using the corresponding _XML_ paths:
+     *     - **hostname**: /instance/hostname
+     *     - **ipAddr**: /instance/ipAddr
+     *     - **port**: /instance/port
+     *
+     *     Additionally XML config contains following properties:
+     *       - **<securePort enabled="true">{port}</securePort>**
+     *       - **<vipAddress>{serviceId}</vipAddress>**
+     *       - **<secureVipAddress>{serviceId}</secureVipAddress>**
+     *       - **<instanceId>{instanceId}</instanceId>**
+     *       - **<dataCenterInfo><name>MyOwn</name></dataCenterInfo>**
+     *
+     *
+     *     **Example in _YAML_:**
+     *     * `https://host:port/servicename` for HTTPS service
+     *
+     *     `baseUrl` is susequently used as a prefix in combination with the following end points relative addresses to construct their absolute URLs:
+     *     * **homePageRelativeUrl**
+     *     * **statusPageRelativeUrl**
+     *     * **healthCheckRelativeUrl**
+     */
     private String baseUrl;
```

Comment:
Maybe also mention that baseUrl is mandatory for the Swagger 2.x configuration

Reply:
baseUrl is not used for the swagger documentation when on-boarding with enabler.